src/mRides-server/Migrations/20170313042519_migration2.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace mRidesserver.Migrations
+{
+    public partial class migration2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
The migrations are auto generated by Asp, therefore they can't be modified. I believe we can add them to .gitignore too since they are used very very occasionaly to update the database. However, for now we can safely ignore the migration files